resolwe/flow/models.py

```diff
@@ -273,22 +272,10 @@ class Meta(BaseModel.Meta):
 
 
 def render_template(template_string, context):
-    """Render template based on Dango template language."""
-    template_headers = [
-        '{% load resource_filters %}',
-        '{% load process_fields %}',
-        '{% load mathfilters %}',
-    ]
-
-    custom_template_tags = getattr(settings, 'RESOLWE_CUSTOM_TEMPLATE_TAGS', [])
-    if not isinstance(custom_template_tags, list):
-        raise KeyError("`RESOLWE_CUSTOM_TEMPLATE_TAGS` setting must be a list.")
-
-    template_headers.extend(
-        ['{{% load {} %}}'.format(template_tag) for template_tag in custom_template_tags]
-    )
+    """Render template using the Jinja expression engine."""
+    from .managers import manager
 
-    return template.Template(''.join(template_headers) + template_string).render(context)
+    return manager.get_expression_engine('jinja').evaluate_block(template_string, context)
```

Comment:
I'm in favour of @kostko's proposal.

1. I think it's cleaner to specify the expression engine for the whole process definition.

2. Being closer to CWL is a nice bonus.